rest-openehr/src/main/java/org/ehrbase/rest/BaseController.java

```diff
@@ -268,7 +273,8 @@ protected MediaType resolveContentType(String acceptHeader, MediaType defaultMed
         MediaType.sortBySpecificityAndQuality(mediaTypes);
         MediaType contentType = mediaTypes.stream()
                 .filter(mediaType -> mediaType.isCompatibleWith(MediaType.APPLICATION_JSON)
-                        || mediaType.isCompatibleWith(MediaType.APPLICATION_XML))
+                        || mediaType.isCompatibleWith(MediaType.APPLICATION_XML)
+                        || mediaType.isCompatibleWith(MediaType.TEXT_PLAIN))
```

Comment:
It seems a lot of callers only specify the media types they produce, but not those they can consume.
Instead, they seem to rely on this method to reject unsupported types.
Therefore, adding TEXT_PLAIN here could be a problem.

Reply:
done